template/packages/app-constants/src/api.constants.ts

```diff
@@ -1,10 +1,10 @@
 export const DATABASE_DOCUMENTS = {
   USERS: 'users',
-  TOKENS: 'tokens',
 };
 
 export const COOKIES = {
   ACCESS_TOKEN: 'access_token',
 };
 
 export const TOKEN_SECURITY_LENGTH = 32;
+export const TOKEN_SECURITY_EXPIRES_IN = '60d';
```

Comment:
let's store this time in seconds for easier converting this time to expiresIn param for cookies